utils/githooks/pre-commit.d/20-flake.sh

```diff
@@ -49,14 +49,14 @@ else
         if [ $TARGET = "HEAD" ]
         then
                 echo "Checking against branch HEAD"
-                git diff HEAD^ -U10 | flake8 --diff
+                git diff HEAD^ -U10 | flake8 --config .flake8 --diff
```

Comment:
What generates the `.flake8` file? Does the command work OK if that file isn't present?

Reply:
It's in git.  The issue seems to be that Jeff is using a distro version of flake8 which is using a different default config file name.